kotlinx-coroutines-core/common/src/CoroutineScope.kt

```diff
@@ -117,13 +123,30 @@ public fun MainScope(): CoroutineScope = ContextScope(SupervisorJob() + Dispatch
 /**
  * Returns `true` when the current [Job] is still active (has not completed and was not cancelled yet).
  *
- * Check this property in long-running computation loops to support cancellation:
+ * A coroutine cancallation [is cooperative](https://kotlinlang.org/docs/cancellation-and-timeouts.html#cancellation-is-cooperative)
```

Comment:
```suggestion
 * Coroutine cancallation [is cooperative](https://kotlinlang.org/docs/cancellation-and-timeouts.html#cancellation-is-cooperative)
```

The article changes the meaning from "coroutine cancellation (as a concept; in general; as a set of rules and behaviors) is cooperative" to "each instance of coroutine cancellation happens in a cooperative manner". Both are true, but I guess the first one is what was meant.